src/overlays/actors/ovl_En_Az/z_en_az.h

```diff
@@ -7,10 +7,48 @@ struct EnAz;
 
 typedef void (*EnAzActionFunc)(struct EnAz*, PlayState*);
 
+#define BEAVER_GET_PARAM1(thisx) (((thisx)->params >> 8) & 0xF)
+#define BEAVER_GET_PARAM2(thisx) ((thisx)->params & 0xFF)
```

Comment:
```suggestion
#define BEAVER_GET_PARAMF00(thisx) (((thisx)->params >> 8) & 0xF)
#define BEAVER_GET_PARAMFF(thisx) ((thisx)->params & 0xFF)
```
I find it easier to deal with if we name the params macros based off the bits they grab when we don't know their function yet.